src/tools/pontoon/teams_projects.md

```diff
@@ -51,12 +51,26 @@ It’s possible to request a project from a Team page. Note that this is a reque
 
 ![Request a project](../../assets/images/pontoon/teams_projects/request_project.png)
 
-Click on `REQUEST MORE PROJECTS`, select the project to add and then click `REQUEST NEW PROJECT` (at least one project needs to be selected for the button to be displayed).
+The button is hidden if all available projects are already enabled for the locale.
+
+Click on `REQUEST MORE PROJECTS`, select the projects to add and then click `REQUEST NEW PROJECT` (at least one project needs to be selected for the button to be displayed).
 
 An email will be sent to Pontoon’s administrators, and the Project manager in charge of the project will act on the request. Please note that:
 * Some projects have a closed list of supported locales, meaning that these projects can’t be requested on Pontoon.
 * Some projects can be requested but may not be enabled for practical restrictions related to the project itself (e.g. lack of support for the locale in iOS).
 
+### Request pretranslation
```

Comment:
Nit: Should this be "Requesting pretranslation" to match the form used in "Requesting a project"? In that case you'd also need to update the link.